docs(readme): Simplified Chinese translation for README

dreamjz opened the pull request:
# Description
Summarize the change.

Fixes #18 

# Testing
N/A

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

asiL-tcefreP commented on README_ZH_CN.md:
```diff
@@ -0,0 +1,111 @@
+# SigLens
+
+SigLens 是一个开源可观测性解决方案，比 Splunk 和 Elastic 高效 **100 倍**。
```
I think maybe "SigLens 是一个开源的，能够解决可观测性问题的方案，它比 ..." will be better

asiL-tcefreP commented on README_ZH_CN.md:
```diff
@@ -0,0 +1,111 @@
+- ElasticSearch 需要太多的机器，集群维护比较困难
+- Grafana Loki 的查询性能较慢
+
+凭借几十年的监控领域经验，我们着手开始构建一个全新的，专为 logs、metrics 和 traces 而设计的，并且具有**零外部依赖**的可观测性数据库。只需一个**单一的二进制文件**，就可以在您的笔记本电脑上运行，处理每天高达 8 TB 的数据。
```
I think maybe "它只需要一个**单一的二进制文件**，就可以在您的笔记本电脑上运行，并且能够每天处理高达 8 TB 的数据。" will be better

asiL-tcefreP commented on README_ZH_CN.md:
```diff
@@ -0,0 +1,111 @@
+
+## 加入我们的 Slack 社区
+在 [Slack](https://www.siglens.com/slack) 上和我们打个招呼吧 👋
+
```
I guess you might need to add two line break characters `<br />` to align with the original document.

asiL-tcefreP commented on README_ZH_CN.md:
````diff
@@ -0,0 +1,111 @@
+```
+
+- 运行 `install_with_docker.sh` 脚本：
+
+```
+./install_with_docker.sh
+```
+
+SigLens 后端独立部署于 UI。 为了允许 UI 连接到后端，可以使用 docker network
+
+```
+wget "https://github.com/siglens/siglens/releases/download/${SIGLENS_VERSION}/server.yaml"
+docker pull siglens/siglens:${SIGLENS_VERSION} 
+mkdir data
+docker run -it --mount type=bind,source="$(pwd)"/data,target=/siglens/data \
+    --mount type=bind,source="$(pwd)"/server.yaml,target=/siglens/server.yaml \
+    -p 8081:8081 -p 80:80 siglens/siglens:${SIGLENS_VERSION}
+```
````
I think you might need to prepend four spaces to each line for indentation, as the original document has done the same.

asiL-tcefreP commented on README_ZH_CN.md:
```diff
@@ -0,0 +1,111 @@
+
+# 特点：
+
+1. 支持多种数据引入格式(Ingestion formats)：Open Telemetry、Elastic、Splunk HEC、Loki
+2. 支持多种查询语言：Splunk SPL、SQL 和 Loki LogQL
+3. 架构简单，易于上手
```
I think using ',' instead of '、' may be better.

asiL-tcefreP commented on README_ZH_CN.md:
```diff
@@ -0,0 +1,111 @@
+
+![Live Tail](./static/assets/readme-assets/live-tail.png)
+
+#### Minion Searches
```
It seems like you forget to translate this line.

asiL-tcefreP reviewed (commented):
Overall, it's good. Just need to address some minor issues.

dreamjz commented:
Thank you for your review and sorry for my mistakes.

### Modification Description
- Modified some sentences based on the review
- Added the missing `<br></br>` and indentation to to align with the original document.
- Use `,` to separate English terms

asiL-tcefreP commented:
Looks pretty good.

dreamjz commented:
### Description 
- Update Chinese doc to align with English doc
